core/tee/tee_svc.c

```diff
@@ -297,6 +315,11 @@ const struct tee_props tee_propset_tee[] = {
 		.data = fw_manufacturer,
 		.len = sizeof(fw_manufacturer)
 	},
+	{
```

Comment:
We could `#ifdef CFG_TA_BTI` this and say that if this property is absent it means it's not supported too.
That should save a small amount of memory on Armv7 platforms.

Reply:
ok